RFS/src/main/java/com/rfs/common/OpenSearchClient.java

```diff
@@ -313,6 +320,12 @@ public Mono<BulkResponse> sendBulkRequest(String indexName, List<BulkDocSection>
                     () -> BulkDocSection.convertToBulkRequestBody(docsMap.values()),
                     error
                 );
+            } else {
+                log.atWarn()
```

Comment:
nit: atError given this is unexpected

Reply:
I'll get this on a follow up PR - thanks!